packages/babel-plugin-transform-block-scoping/test/fixtures/tdz/const-readonly/exec.js

```diff
@@ -0,0 +1,33 @@
+function callNow(f) {
```

Comment:
This test is failing with the old plugin, because the const checks and the tdz checks didn't interact well with each other. Now they are handled together in `validation.ts`.